plan/validator.go

```diff
@@ -450,6 +454,28 @@ func checkColumn(colDef *ast.ColumnDef) error {
 	return nil
 }
 
+func isNowSymFunc(expr ast.ExprNode) bool {
+	if funcCall, ok := expr.(*ast.FuncCallExpr); ok {
+		if funcCall.FnName.L == ast.CurrentTimestamp {
```

Comment:
why not check ast.Now here?

Reply:
```
DefaultValueExpr:
	NowSymOptionFraction | SignedLiteral

NowSymOptionFraction:
	NowSym
	{
		$$ = &ast.FuncCallExpr{FnName: model.NewCIStr("CURRENT_TIMESTAMP")}
	}
|	NowSymFunc '(' ')'
	{
		$$ = &ast.FuncCallExpr{FnName: model.NewCIStr("CURRENT_TIMESTAMP")}
	}
|	NowSymFunc '(' NUM ')'
	{
		$$ = &ast.FuncCallExpr{FnName: model.NewCIStr("CURRENT_TIMESTAMP")}
	}
```

@XuHuaiyu

Reply:
we can add a comment for `isNowSymFunc` to state that default value `NOW()` is transformed to `CURRENT_TIMESTAMP()`

Reply:
ok